README.md

```diff
@@ -1,2 +1,127 @@
+**NOTE:** this is a draft README for not fully implemented yet feature.
+
 # edx-calendar-tab
 Shared calendar for Open edX Courseware
+
+
+Django application that allows course staff enable new "Calendar" tab from CMS (studio).
+From LMS side appears new "Calendar" tab with google calendar associated with current course.
+New calendar events may be created by any enrolled student.
+Calendar may be shared by stuff personal with provided google accounts (with update permission).
+
+1) INSTALLATION
+
+1.1) Install Package
+
+installing manually for evaluation and testing:
+
+    sudo su - edxapp -s /bin/bash
+    . edxapp_env
+    pip install -e git+https://github.com/raccoongang/edx-calendar-tab.git@wowkalucky/event_permissions#egg=edx-calendar-tab
```

Comment:
Change link to `git+https://github.com/raccoongang/edx-calendar-tab.git#egg=edx-calendar-tab`